File: src/tool/spring-boot-generator/output/authority/authority/src/main/java/com/authority/pojo/entity/UserRole.java
package com.authority.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;



@ApiModel("role")
@Getter
@Setter
public class UserRole  {

    @ApiModelProperty("")
    private Integer id;

    @ApiModelProperty("related user ")
    private String userAccount;

    @ApiModelProperty("system id")
    private Integer systemId;

    @ApiModelProperty("create user")
    private String createUser;

    @ApiModelProperty("create time")
    private Date createTime;

}

File: src/tool/spring-boot-generator/output/authority/db/userRole.js
let userRole = {
	name: "userRole",
	description: "role",
	columns: {
		id: {
			name: "id",
			description: "",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		userAccount: {
			name: "userAccount",
			description: "related user ",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		systemId: {
			name: "systemId",
			description: "system id",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		createUser: {
			name: "createUser",
			description: "create user",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "create time",
			nullable: false,
			type: {
				name: "timestamp",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"userAccount",
	"systemId",
	"createUser",
	"createTime"
];

function create() {
  return {};
}

userRole.columnsArray=columnsArray;
userRole.create=create;
module.exports=userRole;